### PR TITLE
Add a .env file to codebase

### PR DESCRIPTION
**Pull request by ezrogha**
#### What does this PR do?
Adds a .env file to the codebase

#### How should this be manually tested?
 - Add variable to the .env file e.g `baseURL='localhost:8080`
 - Access variable by using `process.env.VARIABLE_NAME` e.g process.env.baseURL

#### What are the relevant pivotal tracker stories?
[#165972826](https://www.pivotaltracker.com/story/show/165972826)

### CODE REVIEW

**Review by dannylwe — APPROVED** [review]
LGTM

File: .env-sample [hunk 1]
```diff
@@ -0,0 +1 @@
+baseURL = '';
```
Comment (adesege) [inline .env-sample]: Remove the space before and after the `=` sign. Also, remove the semi colon and single quote. `dotenv` will read it as part of the value.

It should be `baseURL=` instead

File: src/actions/signupSigninActions.js [hunk 2]
```diff
@@ -1,10 +1,9 @@
 import { notify } from 'react-notify-toast';
 import axios from 'axios';
-import config from '../../dist/config';
 
 export const signUpUser = (user, history) => () => {
   axios
-    .post(`${config.baseURL}/register/`, user)
+    .post(`${process.env.baseURL}users/register/`, user)
```
Comment (adesege) [inline src/actions/signupSigninActions.js]: Add a trailing `/` before `users/register`. Fix in other places as well

**Review by adesege — CHANGES_REQUESTED** [review]

**Review by felixkiryowa — APPROVED** [review]
Good work Keep it up . It's good to go

File: src/actions/signupSigninActions.js (hunk 2 above)
Comment (ezrogha) [reply src/actions/signupSigninActions.js]: Thanks for that feedback @adesege,
I have implemented it

File: .env-sample (hunk 1 above)
Comment (ezrogha) [reply .env-sample]: Thanks for that observation @adesege I had missed that,
Feedback implemented

**Comment by adesege** [comment]
Did you test all the features to be sure they are working and there's no breaking changes? @ezrogha

**Comment by ezrogha** [comment]
> Did you test all the features to be sure they are working and there's no breaking changes? @ezrogha

Yes @adesege It works perfectly, I will also have to set the variable remotely as well

**Review by adesege — APPROVED** [review]

**Pull request merged**